GitUI/UserControls/RevisionGrid/RevisionGridControl.cs

```diff
@@ -1111,6 +1111,7 @@ void OnRevisionReadCompleted()
                     {
                         await this.SwitchToMainThreadAsync();
 
+                        _gridView._revisionGraph.LoadingFinished = true;
```

Comment:
This looks like we're breaking encapsulation...

Reply:
https://github.com/gitextensions/gitextensions/commit/cd52038252b67de783cbafe6fda5d5ebd57dbb8b
Or should `RevisionGridControl.OnRevisionReadCompleted` call `_gridView._revisionGraph.LoadingCompleted()` directly?